static/app/views/performance/transactionSummary/transactionOverview/durationChart/index.tsx

```diff
@@ -142,16 +144,24 @@ function DurationChart({
         referrer="api.performance.transaction-summary.duration-chart"
         queryExtras={queryExtras}
       >
-        {({results, errored, loading, reloading, timeframe: timeFrame}) => (
-          <Content
-            series={results}
-            errored={errored}
-            loading={loading}
-            reloading={reloading}
-            timeFrame={timeFrame}
-            {...contentCommonProps}
-          />
-        )}
+        {({results, errored, loading, reloading, timeframe: timeFrame}) => {
+          // Function arguments are superflous and take up space in the chart
+          // labels, so remove them.
+          const series = results?.map(result => ({
+            ...result,
+            seriesName: `${parseFunction(result.seriesName)?.name}()`,
```

Comment:
I sometimes just make a named function to explain the `why` for something like this. eg. `function stripSeriesForLegend()`

Reply:
I like it :+1: Changed!